x-pack/legacy/plugins/maps/public/layers/styles/vector/components/legend/vector_style_legend.js

```diff
@@ -35,7 +35,7 @@ export class VectorStyleLegend extends Component {
     const rowDescriptors = rows.map(row => {
       return {
         label: row.label,
-        range: row.range,
+        range: row.meta,
```

Comment:
How about renaming `range` here as well

Reply:
I'd wait on this, since this component explicitly expects a range (it's implemented to only expect a range). Will put up new refactor that is going to push this down to the dyn-prop, which will remove the need for this.